### DISCUSSION TITLE
how should I run graphrag 0.5.0 with ollama local models now that 1.0.1 has released?

### DISCUSSION THREAD

**Discussion by retgreen98** [microsoft/graphrag#1537] [Q&A]
First, many thanks to developers of this project! Excellent work!
I'm trying to run graphrag and I successed before following this tutorial:https://blog.csdn.net/EEEric_/article/details/143951187
it worked perfectly. Now I want to add more document to to the training text and run it again. However ,the tutorial is no longer working:it tells us to change contents of graphrag/llm/openai/openai_embeddings_llm.py, but in new version of graphrag the file doesn't exist......
Pleeeeeeease help me.
Now I'm trying to git the old version of graphrag, no sure if it'll work though.

**Comment by retgreen98** [comment]
gitting the old version of graphrag actually works!

**Reply by hiitskon** [reply]
@retgreen98 Could you share the version of graphrag you use?
Thank you!

**Reply by retgreen98** [reply]
0.5.0
